src/main/java/uk/gov/companieshouse/charges/data/requests/ChargesRequest.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.companieshouse.charges.data.requests;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.api.charges.InternalChargeApi;
+
+public class ChargesRequest {
+    @JsonProperty("charges")
```

Comment:
I dont think we need this wrapper request? the payload should InternalChargesApi itself

Reply:
Wrapper removed